SkimTools/SkimPhiPi/test/run_Data2024_DsPhiPiSkimAndTree_cfg.py

```diff
@@ -0,0 +1,78 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('DsPhiPiSkim')
+
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.load("TrackingTools/TransientTrack/TransientTrackBuilder_cfi")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load("TrackingTools.TransientTrack.TransientTrackBuilder_cfi")
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+#process.load('DsPhiPiTreeMaker.DsPhiPiTreeMaker.DsPhiPiSkimAOD_cff')
+#process.load('DsPhiPiTreeMaker.DsPhiPiTreeMaker.DsPhiPiMuMuPi_BParking_cff')
+process.load('SkimTools.SkimPhiPi.DsPhiPiMuMuPi_miniAOD_cff')
+
+#process.GlobalTag.globaltag = '102X_dataRun2_v13' #data_rereco 2018 A-C 
+#process.GlobalTag.globaltag = '124X_dataRun3_Prompt_v4' #data_reco 2022 B-D
+#process.GlobalTag.globaltag = '124X_dataRun3_Prompt_v10' #data_reco 2022 G
+process.GlobalTag.globaltag = '140X_mcRun3_2024_realistic_v26'
```

Comment:
This is the MC global Tag, the data one would be 40X_dataRun3_v20. But when processing from MiniAOD, very little is recomputed for which conditions are read from the GT, so it shouldn't matter. Still, would be good to update here.